packages/lib/bookings/findQualifiedHosts.ts

```diff
@@ -114,42 +114,31 @@ export const findQualifiedHosts = async <
     };
   }
 
-  const hostsAfterContactOwnerMatching = applyFilterWithFallback(
```

Comment:
Finding the contact owner should be done from the actual RR hosts which is after segment filtering. That's also how it was before